src/pages/[platform]/build-a-backend/server-side-rendering/nuxt/index.mdx

```diff
@@ -0,0 +1,571 @@
+              ),
+            getCurrentUser: () =>
+              runWithAmplifyServerContext(
+                amplifyConfig,
+                libraryOptions,
+                (contextSpec) => getCurrentUser(contextSpec)
+              )
+          },
+          Storage: {
+            list: (input: ListPaginateInput) =>
+              runWithAmplifyServerContext(
+                amplifyConfig,
+                libraryOptions,
+                (contextSpec) => list(contextSpec, input)
+              )
+          },
+          GraphQL: {
+            client: {
+              graphql: <
+                FALLBACK_TYPES = unknown,
+                TYPED_GQL_STRING extends string = string
+              >(
+                options: GraphQLOptionsV6<FALLBACK_TYPES, TYPED_GQL_STRING>,
+                additionalHeaders?: Record<string, string>
+              ) =>
+                runWithAmplifyServerContext<
+                  GraphQLResponseV6<FALLBACK_TYPES, TYPED_GQL_STRING>
+                >(amplifyConfig, libraryOptions, (contextSpec) =>
+                  gqlServerClient.graphql(
+                    contextSpec,
+                    options,
+                    additionalHeaders
+                  )
+                )
+            }
+          }
+        }
+      }
+    };
+  }
+});
+```
+
+</Accordion>
+
+#### Usage Example
+
+Configure Amplify in `./app.vue`:
+
+```ts
+
+<script setup>
+import { Amplify } from "aws-amplify";
+import config from "./src/amplifyconfiguration.json";
+
+// Amplify.configure() only needs to be called on the client side
+if (process.client) {
+  (function configureAmplifyForClient() {
+    console.log("Configuring Amplify on the client side");
+    Amplify.configure(config, { ssr: true });
+  })();
+}
+</script>
+
+<template>
+  <NuxtPage />
+</template>
+
+```
+
+Using the Storage `list` API in `~/pages/storage-list.vue`:
+
+```ts
+
+// `useAsyncData` and `useNuxtApp` are Nuxt composables
+// `$Amplify` is generated by Nuxt according to the `provide` key in the plugins we've added above
+<script setup lang="ts">
+const { data, error } = useAsyncData(async () => {
+    const listResult = await useNuxtApp().$Amplify.Storage.list({
+      options: {accessLevel: 'guest'}
+    });
+    return listResult.items;
+})
+</script>
+
+<template>
+  <pre>{{ data }}<pre>
+</template>
+
+```
+
+This page can be rendered on both the client side and server sides by default. `useNuxtApp().$Amplify` will pick up the correct implementation of `01.amplify-apis.client.ts` and `01.amplify-apis.server.ts` to use, depends on the runtime.
+
+<Callout warning>
+
+Only a subset of Amplify APIs are usable on the server side; depending on your specific use case, `amplify-apis.client` and `amplify-apis.server` may grow out of sync due to API parity. In this case, you may need to guard your API calls to ensure an API is available. E.g., you can use `if (process.server)` to guard an API call that only happens on the server side.
```

Comment:
Did you mean to say `if (process.client)` so that a client-only can't method won't execute on server? If so, recommend rewording to:

> E.g., you can use `if (process.client)` to ensure that a client-only API isn't executed on the server.